amy/autoemails/models.py

```diff
@@ -222,6 +223,12 @@ def build_email(
             body=body.text,
         )
         msg.attach_alternative(body.html, "text/html")
+        # https://anymail.readthedocs.io/en/stable/sending/templates/#batch-sending-with-merge-data
+        # https://anymail.readthedocs.io/en/stable/sending/templates/#anymail.message.AnymailMessage.merge_data
+        # When set the recipients in the two column each get an individual message
```

Comment:
I don't understand. What's `two column` in this case?

Reply:
Oh sorry I meant every recipient gets an individual message.